README.md

```diff
@@ -289,10 +310,19 @@ page.invokeMethod('evaluate', function(selector) {
 });
 ```
 
+#### `page#invokeMethod('setContent', expectedContent, expectedLocation)`
+
+Set content and/or URL address of the page, without waiting for additional elements to load.
+
+If `page#invokeMethod('render', ...)` is called right after, it might not execute properly (see `page#loadContent` notes).
+
+#### `page#invokeMethod('render', filename, [{format, quality}])`
```

Comment:
Perhaps we should fix it so that people don't think they need to call `invokeMethod`